aiosmtpd/smtp.py

```diff
@@ -329,11 +329,8 @@ def _set_rset_state(self):
         await self.push(status)
 
     async def smtp_NOOP(self, arg):
-        if arg:
-            await self.push('501 Syntax: NOOP')
-        else:
-            status = await self._call_handler_hook('NOOP')
-            await self.push('250 OK' if status is MISSING else status)
+        status = await self._call_handler_hook('NOOP', arg)
```

Comment:
Previously the `handle_NOOP()` hook didn't receive the `arg`. This is a change in API, so the documentation must be updated to reflect this.